### PR TITLE
adds stdeb and its dependencies

### PR DESCRIPTION
**Pull request by kzidane**
@dmalan any concerns about this?

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -19,6 +19,7 @@ RUN apt-get update && \
         build-essential \
         clang-3.6 \
         curl \
+        dh-make \
```
Comment (dmalan) [inline Dockerfile]: Do we need this even with `fpm`?

File: Dockerfile [hunk 2]
```diff
@@ -39,10 +40,10 @@ RUN apt-get update && \
         php5-gmp \
         php5-intl \
         php5-mcrypt \
-        python \
+        python-all \
```
Comment (dmalan) [inline Dockerfile]: What does `-all` add?

Comment (kzidane) [inline Dockerfile]: Practically nothing. [`python-all`](http://packages.ubuntu.com/trusty/python-all) depends on `python` and `python2.7`, which is a [dependency](http://packages.ubuntu.com/trusty/python) for `python` itself. Same thing for `python3-all`.

Not having them installed results an error when building with `stdeb`, listing them as dependencies, though!

**Comment by kzidane** [comment]
Closing as we're going to follow the "`VERSION` file" approach per Slack!

**Pull request closed without merging**